build-scripts/puddle-conf/atomic_openshift-4.1.el8.conf

```diff
@@ -0,0 +1,17 @@
+[puddle]
+type = simple
+rootdir = /mnt/rcm-guest/puddles/RHAOS
+mashroot = /mnt/rcm-guest/puddles/RHAOS/mash
+brewroot = file:///mnt/redhat/brewroot
+topurl = http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/
+name = AtomicOpenShift.el8
```

Comment:
So the full repo url will be something like... http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift.el8/4.1/latest/ Feel like it might be slightly confusing given the base repo is http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift/4.1 but no real objections whichever way the naming scheme ends up

Reply:
(i.e. there is no `el7` in the other)

Reply:
You're right, the other doesn't have a `.el7` part, but we can't change it right now because there are a lot of systems depending on that URL.

Reply:
Yeah its fine, either way if the job files look good I'm :+1: